django/db/models/fields/__init__.py

```diff
@@ -2408,3 +2386,150 @@ def formfield(self, **kwargs):
         }
         defaults.update(kwargs)
         return super(UUIDField, self).formfield(**defaults)
+
+
+class DateTransform(Transform):
+    extract_type = None
```

Comment:
could `extract_type` default to `lookup_name`? not sure if it's different for any of the transforms we have so far.